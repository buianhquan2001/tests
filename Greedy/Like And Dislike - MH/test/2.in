5
34 86 56
73 53 24
28 53 13
12 37 49
10 62 100
