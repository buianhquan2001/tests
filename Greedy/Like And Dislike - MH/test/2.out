Unknown
Unknown
Dislike
Unknown
Unknown
